apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/core/LogLevel.java

```diff
@@ -19,5 +19,5 @@
 package org.apache.skywalking.apm.agent.core.logging.core;
 
 public enum LogLevel {
-    DEBUG, INFO, WARN, ERROR, OFF
+    TRACE, DEBUG, INFO, WARN, ERROR, OFF
```

Comment:
Please help with updating `logging.level` in the agent setup doc. Add all available values in the documentation.